engine/src/main/resources/META-INF/camunda-renames.properties

```diff
@@ -23,5 +23,6 @@
 com.fasterxml.jackson.jaxrs.*=com.fasterxml.jackson.jakarta.rs
 org.camunda.bpm.application.impl.EjbProcessApplication=org.camunda.bpm.application.impl.JakartaEjbProcessApplication
 org.camunda.bpm.application.impl.ServletProcessApplication=org.camunda.bpm.application.impl.JakartaServletProcessApplication
+ServletProcessApplication=JakartaServletProcessApplication
```

Comment:
@tmetzke What do you think, could this cause any issues in other modules that use the transform plugin?
I guess CI will find it out.